Gemfile

```diff
@@ -48,6 +48,10 @@ gem 'rails_admin', '~> 1.2'
 # gem 'cancancan', '~>2.0'
 
 gem 'sidekiq'
+gem 'sidekiq-scheduler'
```

Comment:
Are we using this ?

Reply:
No its not required ill update the pr soon